src/Symfony/Component/Security/Guard/AbstractGuardAuthenticator.php

```diff
@@ -23,7 +23,7 @@
 abstract class AbstractGuardAuthenticator implements AuthenticatorInterface
 {
     /**
-     * {@inheritdoc}
+     * @deprecated since version 3.4, to be removed in 4.0
```

Comment:
this one should be removed, as it could create confusion when static analyzer are looking at code overriding the method. The deprecated thing is relying on that implementation of the method instead of implementing in the child class.

Reply:
Good point. Would it make sense to consider `@inheritdoc` as a false-positive, like in this case? PS: I'll revert.